dev/tidb-in-kubernetes/faq.md

```diff
@@ -78,3 +78,23 @@ TiDB Operator 尚不支持自动编排 TiSpark。
     ```shell
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## 部署 TiDB 集群时调度失败是什么原因？
+
+TiDB Operator 调度 Pod 失败的原因可能有三种情况：
+
+* 资源不足的问题，这时 Pod 一直阻塞在 `Pending` 状态，这块我们在集群故障诊断中有详细说明，请参考：[集群故障诊断](/dev/tidb-in-kubernetes/troubleshoot.md)
+
+* 部分 Node 被打了 `taint`，导致 Pod 无法调度到对应的 Node 上。请参考：[taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+
+* 调度错误，这时 Pod 一直阻塞在 `ContainerCreating` 状态。这种情况发生时请检查 Kubernetes 集群中是否部署过多个 TiDB Operator。重复的 TiDB Operator 自定义调度器的存在，会导致同一个 Pod 的调度周期不同阶段会分别被不同的调度器处理，从而产生冲突。
```

Comment:
For TiKV and PD, the Pod may be pending due to HA policy.
Maybe the troubleshoot.md should be updated to cover this case.

Reply:
@anotherrachel FYI. Daniel will update `troubleshoot.md` in a new PR.